src/js/components/Box/stories/Simple.js

```diff
@@ -17,16 +17,16 @@ export const SimpleBox = () => (
     <Box
       pad="large"
       align="center"
-      background={{ color: 'light-2', opacity: 'strong' }}
+      background={{ color: 'light-4', opacity: 'strong' }}
       round
       gap="small"
     >
       <Attraction size="large" />
       <Text>Party</Text>
-      <Anchor href="" label="Link" />
+      <Anchor href="" label="Link" color="blue-4" />
```

Comment:
```suggestion
      <Anchor href="" label="Link" color="dark-1" />
```

`blue-4` is not a `color` tha
<img width="162" alt="Screen Shot 2023-10-11 at 10 12 33 AM" src="https://github.com/grommet/grommet/assets/42451602/bc7bfd23-23ee-436b-b1aa-e1079af5f368">
t we have declared can we use `dark-1`